src/set_theory/game/basic.lean

```diff
@@ -64,94 +60,61 @@ by { rintro ⟨x⟩ ⟨y⟩ ⟨z⟩, apply pgame.le_trans }
 theorem le_antisymm : ∀ x y : game, x ≤ y → y ≤ x → x = y :=
 by { rintro ⟨x⟩ ⟨y⟩ h₁ h₂, apply quot.sound, exact ⟨h₁, h₂⟩ }
 
-/-- The relation `x < y` on games. -/
--- We don't yet make this into an instance, because it will conflict with the (incorrect) notion
--- of `<` provided by `partial_order` later.
```

Comment:
Haven't you diregarded this comment by making it an instance?

Reply:
I believe I have. I don't think it's an issue though, since `partial_order` was made into a `def` rather than an `instance`.

Reply:
In that case, can you add a comment on this instance that indicates that it is incompatible with the `has_lt` instance from `game.partial_order`, but as it is the "canonical" definition, `game.partial_order` is made the non-instance.